crates/hdk/src/link.rs

```diff
@@ -113,7 +113,13 @@ pub fn delete_link(address: ActionHash) -> ExternResult<ActionHash> {
     })
 }
 
-/// Returns all links that reference a base entry hash, optionally filtered by tag.
+/// Returns all links that reference a base entry hash, optionally filtered by link type and tag.
+///
+/// Type can be filtered by providing a variant of the link types, or a range of them. To get links of
```

Comment:
Note this will only get all links that are of types that are in this zomes dependencies.

Reply:
> this zomes dependencies

@freesig What does this mean? Let's say I have a function in my coordinator zome that does `get_links`, and I have no other coordinator zomes in the DNA. What does it refer to?

Reply:
I think the dependency referred to here is an integrity zome dependency.